Beschrijver.hs

```diff
@@ -0,0 +1,68 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
Ik denk dat dit ook niet nodig is, want je gebruikt in deze module alleen String's, geen Text's